Tests/setup_test_datacube.sh

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env bash
 set -ex
+set -o pipefail
```

Comment:
What do these lines do? Can we add a code comment to make it clear for future readers?